native/screens/SingleProject.jsx

```diff
@@ -29,6 +25,18 @@ export const SingleProject = ({ navigation, route }) => {
     setLoaded(true);
   }, [route]);
 
+  let rolesSought;
+  if (project.designer_count && project.engineer_count) {
+    rolesSought = 'UX Designer, Software Engineer';
+  } else if (project.designer_count && !project.engineer_count) {
+    rolesSought = 'UX Designer';
+  } else if (!project.designer_count && project.engineer_count) {
+    rolesSought = 'Software Engineer';
+  } else {
+    rolesSought = null;
```

Comment:
For some reason this is rendering as "Seeking null" when I tested this as a non-member/non-owner user.
Also when I log in as different users, the rolesSought eternally becomes equal to null?

Should rolesSought be handled as state local to the component so it's sure to update with new users?

Reply:
Updated this to add a condition to check the `engineer_count` and `designer_count` properties of the project and not to render the seeking info if they are both set to `0` even if the `seeking` property is set to `true`.